### PR TITLE
[TESTING] mobile/ci: Increase timeouts

### PR DESCRIPTION
**Pull request by phlax**
Partial fix for #26912 

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
this should fix most of the mobile ci issues ive been hitting (a lot) when trying to run release ci

unless there is a risk of these jobs hanging, i dont see so much downside to just being more liberal with timeouts

we can probably test the current state of play by running a PR which rings all the mobile bells, and doesnt cache much - i suspect that PR would need at least these timeouts

**Comment by phlax** [comment]
watching the ci flake immediately (unrelatedly) suggests we need to add some caching similar to what we have done recently for envoy

**Comment by phlax** [comment]
> unless there is a risk of these jobs hanging, i dont see so much downside to just being more liberal with timeouts

hmm, i see, the jobs _do_ appear to be hanging

File: .github/workflows/mobile-android_build.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
     name: java_helloworld
     needs: androidbuild
     runs-on: macos-12
-    timeout-minutes: 50
```
Comment (alyssawilk) [inline .github/workflows/mobile-android_build.yml:48]: yeah these could actually be shortened, as they don't actually take that long

**Comment by phlax** [comment]
closing this as probably unhelpful

**Pull request closed without merging**